src/status_im/subs/wallet/swap.cljs

```diff
@@ -21,9 +21,11 @@
 
 (rf/reg-sub
  :wallet/swap-asset-to-pay-networks
- :<- [:wallet/swap-asset-to-pay]
- (fn [token]
-   (let [{token-networks :networks} token
+ (fn []
+   [(rf/subscribe [:wallet/swap-asset-to-pay])
+    (rf/subscribe [:wallet/current-viewing-account-tokens-filtered])])
```

Comment:
Why are you subscribing this way?

Reply:
@ajayesivan since we aren't taking any values in the signal function we can use the syntax with `:<-` twice.
i.e.:
```clojure
(rf/reg-sub 
 :my-sub
 :<- [:a]
 :<- [:b]
 (fn [[a b]]
  ,,,))
```